src/main/java/Main.java

```diff
@@ -37,135 +43,176 @@
 
     };
 
+    private int tableOfIMP[][] = {{20, 40, 1}, {50, 80, 2}, {90, 120, 3}, {130, 160, 4}, {170, 210, 5},
```

Comment:
Wydaje mi się, iż fajnie by było stworzyć jakąś klasę do przechowywania danych z tej tabeli. Klasa ta miałaby 3 pola, coś w stylu `int lowestValue`, `int highestValue`, `int points`, czy coś w tym stylu. Wtedy czytasz dane z tej tabeli i tworzysz listę obiektów tej klasy. Takie sprawdzanie jak masz w 203 linijce powinno być w jakiejś metodzie tej klasy. Ogólnie te tabele pewnie w przyszłości będa przerabiane na klasy, więc neich to bedzie pierwszy krok.